pkg/cli/instances.go

```diff
@@ -42,7 +42,13 @@ func Instances(rack sdk.Interface, c *stdcli.Context) error {
 	t := c.Table("ID", "STATUS", "STARTED", "PS", "CPU", "MEM", "PUBLIC", "PRIVATE")
 
 	for _, i := range is {
-		t.AddRow(i.Id, i.Status, common.Ago(i.Started), fmt.Sprintf("%d", i.Processes), common.Percent(i.Cpu), common.Percent(i.Memory), i.PublicIp, i.PrivateIp)
+		cpuPercent, memPercent := common.Percent(i.Cpu), common.Percent(i.Memory)
+		// for rack version 3, if allocatable metrics available
```

Comment:
#nit you can remove this comment as this repo only contains the code for v3 racks.

Reply:
removed it